hi-ml-cpath/testhisto/testhisto/utils/test_plots_utils.py

```diff
@@ -128,6 +128,10 @@ def test_save_conf_matrix_integration(tmp_path: Path) -> None:
     file = Path(tmp_path) / "normalized_confusion_matrix_foo.png"
     assert file.exists()
 
+    save_confusion_matrix(results, class_names, tmp_path, stage='foo', normalize=None)
+    file = Path(tmp_path) / "confusion_matrix_foo.png"
+    assert file.exists()
+
```

Comment:
Maybe add some stored pngs to the tests here? Will be helpful also do see visually if the normalization is really done the right way.